docs/advanced-topics/linear-probe.md

```diff
@@ -0,0 +1,61 @@
+(linear-probe)=
+# {octicon}`pin` Linear Probe
+
+## Why freezing?
+
+Depending on your task and the amount of training data,
+sometimes it is unnecessary to tune the entire model.
+In these cases,
+freezing the weights of the pre-trained model and fine-tuning specific layers is sufficient enough.
+Besides, freezing can reduce the training time considerably.
+
+Finetuner allows you to fine-tune a Linear Probe (or Projection Head) easily.
+
+```{warning}
+Currently, we only allow you to freeze layers for image-to-image search tasks.
```

Comment:
I'm a little confused why this is true. Is it just that some kinds of models are supported?

Reply:
yes we only enabled it for CNNs, not transformers. It is supported inside Core, not Client

Reply:
TODO:  make it clear why only image-to-image